Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -188,16 +188,27 @@ public Camera OutputCamera
         {
             get
             {
-                if (m_OutputCamera == null && !Application.isPlaying)
+                if (m_OutputCamera == null || !Application.isPlaying)
```

Comment:
@glabute Do you know what's the reason behind this check here?

Reply:
If m_OutputCamera is null on purpose, we don't want to be wasting time searching for components every frame.  In the editor, it's ok, we want to auto-detect, but not if playing.  

What was the reasoning behind changing the && to || ?